### PR TITLE
Results page

### PR DESCRIPTION
**Pull request by franzmoro**

### CODE REVIEW

File: frontend/src/js/Header/index.jsx [hunk 1]
```diff
@@ -9,8 +9,12 @@ class Header extends React.Component {
...
         <Link to='/'><h1>Amazon 2.0 Home Page</h1></Link>
         <header style={styles}>
-          <SearchBar submitHandler={this.props.search} categorySelect={this.props.categorySelect} handleChange={this.props.handleChange} />
-          <Nav home={'/home'} browse={'/search'} checkout={'/checkout'} basket={'/basket'}/>
+          <SearchBar
+            submitHandler={this.props.search}
+            categorySelect={this.props.categorySelect}
+            handleChange={this.props.handleChange}
+            setResultsState={this.props.setResultsState} />
+          <Nav home={'/home'} browse={'/search'} checkout={'/payment'} basket={'/basket'}/>
```
Comment (katbow) [inline frontend/src/js/Header/index.jsx]: checkout here needs to stay as '/checkout' I think. The payment was not working when checkout={'/payment'}

File: frontend/src/js/amazon.jsx [hunk 2]
```diff
@@ -5,70 +5,44 @@ import PaymentStatus from './PaymentStatus/index.jsx'
 import SearchResults from './SearchResults/index.jsx'
 import ProductPage from './ProductPage/ProductPage.jsx'
 import Basket from './BasketEntry/index.jsx'
+import { browserHistory, Router, Route, IndexRoute } from 'react-router'
 import Checkout from './Checkout/index.jsx'
```
Comment (katbow) [inline frontend/src/js/amazon.jsx:10]: I think PaymentStatus needs to be imported here from ./PaymentStatus/index.jsx - it's used below in the routes but not defined up here.

File: frontend/src/js/Header/index.jsx (hunk 1 above)
Comment (franzmoro) [inline frontend/src/js/Header/index.jsx]: ok, sorted

File: frontend/src/js/amazon.jsx (hunk 2 above)
Comment (franzmoro) [inline frontend/src/js/amazon.jsx:10]: it's on line 4

Comment (katbow) [inline frontend/src/js/amazon.jsx:10]: Oh my bad, I was looking on the train and had a thought that it was possibly on one of the lines above heh

**Comment by katbow** [comment]
Before this is merged, would someone be able to double check that payment runs through okay? I couldn't manage to get it working earlier, even when @tu6619 had it working at the same time from the same branch :confused: . I had the config.env file but no dice :game_die:

**Comment by eliasmalik** [comment]
@katbow payments looks like it's working for me now. Give it a go.

**Comment by katbow** [comment]
@eliascodes @franzmoro works great :+1:

**Pull request merged**